common/persistence/data_manager_interfaces.go

```diff
@@ -968,7 +909,10 @@ type (
 	// GetReplicationTasksFromDLQRequest is used to get replication tasks from dlq
 	GetReplicationTasksFromDLQRequest struct {
 		SourceClusterName string
-		GetReplicationTasksRequest
+		ReadLevel         int64
+		MaxReadLevel      int64
+		BatchSize         int
+		NextPageToken     []byte
```

Comment:
nit: should we keep it? The benefit of composition is if we add more fields, we don't need extra changes. For GetReplicationTasksFromDLQRequest, it extends the GetReplicationTasksRequest by adding a source cluster name field.

Reply:
GetReplicationTasksRequest is deleted because GetReplicationTasks method is removed. But that's a good point, I think I should let GetReplicationTasksFromDLQRequest reuse GetHistorytasksRequest instead.